services/core-web/src/components/common/RenderAutoSizeField.js

```diff
@@ -16,37 +16,60 @@ const propTypes = {
   meta: PropTypes.objectOf(PropTypes.any).isRequired,
   disabled: PropTypes.bool,
   minRows: PropTypes.number,
+  maximumCharacters: PropTypes.number,
 };
 
 const defaultProps = {
   placeholder: "",
   label: "",
   disabled: false,
   minRows: 3,
+  maximumCharacters: 300,
```

Comment:
This component is used in quite a few places, I'm not 100% sure we want to enable character limiting by default. Have we done any investigation into that?

Reply:
Nice catch, I will refactor.